src/compoz.ts

```diff
@@ -267,6 +271,40 @@ export class Compoz {
 		this.elBSendImg.src = svgSendDisable
 	}
 
+	getLastValue(arr: string[]) {
+		return arr[arr.length - 1]
+	}
+
+	resetHistoryChanges() {
+		this.historyChanges = []
+		this.historyChanges.push("")
+		this.redoChanges = []
+		this.oldChange = ""
+	}
+
+	undoContent() {
+		if (this.historyChanges.length === 1) {
+			return
+		}
+		this.redoChanges.push(this.oldChange)
+		this.historyChanges.pop()
+		this.oldChange = this.getLastValue(this.historyChanges)
```

Comment:
I think this should be `this.oldChange = this.historyChanges.pop()`

Reply:
Assume that we have three items in `historyChanges`: [a, b,c] and redoChanges is empty, and oldContent is `d` Calling `undo()` will do this,

```
redoChanges.push(this.oldChange) // redoChanges = [d]
historyChanges.pop() //  historyChanges = [a,b]
this.oldChange = this.getLastValue(this.historyChanges) // oldChange = b, it should be c
...
```
Is it correct? did i miss something?